src/components/FeatureViews/Scheduler/EditSessionView.js

```diff
@@ -33,6 +37,9 @@ const EditSessionView = ({course, session, editSelection}) => {
 		[dispatch]
 	);
 	const history = useHistory();
+	const location = useLocation();
+	const { course, session } = location.state;
```

Comment:
You can't use `location.state` to get this - it won't load the course/session if the user tries to open this link without pressing a certain button.

Reply:
@NeilAllavarpu we're going to open a new PR for this - @jsflca2012 will be migrating session view to graphQL